### ISSUE TITLE
enable "allow backup"

### ISSUE THREAD

**Issue by thrylosthyra7** [kiwibrowser/src.next#978]
**Is your feature request related to a problem? Please describe.**
a lot of apps allow to be backed us via adb. but not kiwi browser. could you enable it? it's easy and it will make our life easier..

**Describe the solution you'd like**
enable "allow backup"

**Comment by GhettiGuru** [comment]
Yes again, none of the cookie exporting functions seem to work with any of the extensions and you're unable to save at least you're you know basic login sessions and stuff and in my case I'm not going to be able to recover them so this is going to be necessary. With root this would be achievable in several different ways but... I have to factory reset to root and I cannot do that without recovering some of these sessions. As noted in my other post if you work around I can think of would be like it's open source brothers and sisters bromite and the new cromite they both allow, well originally firewalling off the Google login native inside browser for syncing, in the settings you will find a place to enable it that will allow you to directly log in and browser and cloud synchronize your settings. Including history, what if this would be out of the scope of your project, have you considered like I mentioned, adding like Mozilla device with all of theirs and they're open sores full extension supporting variant, ice raven, a kiwi based separate account and login to synchronize to specific for your browser that does the same functionality
